sklearn/utils/tests/test_stats.py

```diff
@@ -1,23 +0,0 @@
-from sklearn.utils.testing import assert_array_equal
```

Comment:
I think you should keep this test and add ignore_warnings like you did for other deprecated wrappers